src/DynamoCoreWpf/Controls/AddonsTreeView.xaml.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Dynamo.Search.SearchElements;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+
+namespace Dynamo.UI.Controls
+{
+    /// <summary>
+    /// Interaction logic for AddonsTreeView.xaml
+    /// </summary>
+    public partial class AddonsTreeView : UserControl
+    {
+        public AddonsTreeView()
+        {
+            InitializeComponent();
+        }
+
+        private void OnEditClick(object sender, RoutedEventArgs e)
```

Comment:
If `OnEditClick` is not needed, please removed it and also the corresponding entry in `*.xaml` if there's any.